tests/text/test_wer.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from jiwer import compute_measures
+
+from torchmetrics.functional.text.wer import wer
+from torchmetrics.text.wer import WER
+
+
+@pytest.mark.parametrize(
+    "hyp,ref,score",
+    [("hello world", "hello world", 0.0), ("hello world", "Firwww", 1.0)],
+)
+def test_wer_same(hyp, ref, score):
+    metric = WER()
+    metric.update(hyp, ref)
+    assert metric.compute() == score
+
+
+@pytest.mark.parametrize(
+    "hyp,ref,score",
+    [(["hello world"], ["hello world"], 0.0), (["hello world"], ["Firwww"], 1.0)],
+)
+def test_wer_functional(hyp, ref, score):
+    assert wer(ref, hyp) == score
+
+
+@pytest.mark.parametrize(
+    "hyp,ref,score",
+    [(["hello world"], ["hello world"], 0.0), (["hello world"], ["Firwww"], 1.0)],
+)
+def test_jiwer(hyp, ref, score):
+    assert compute_measures(ref, hyp)["wer"] == score
```

Comment:
```suggestion
```

Reply:
I am a little confused by what we have to do here?

Reply:
Pls check how the BLUE is tested
https://github.com/PyTorchLightning/metrics/blob/master/tests/text/test_blue.py